functions/go/fix/fixpkg/fix.go

```diff
@@ -748,3 +752,11 @@ func schemaUsingField(object *yaml.RNode, field string) (*spec.Schema, error) {
 
 	return &sc, nil
 }
+
+// GenerateShortHash returns the short hash for input string
```

Comment:
is this to prevent collision in case there are multiple instances of the function? Consider only appending the hash in case of conflict.

Reply:
Actually, conflicts happen if there are 2 function configs with same name. I think it is highly unlikely. We need not optimize for that. So to keep it simple, I just used resource name.